smartsim/_core/_cli/build.py

```diff
@@ -67,8 +67,6 @@ def check_py_onnx_version(versions: Versioner) -> None:
             f"Python {py_version.major}.{py_version.minor}. "
             "Instead consider using Python 3.8 or 3.9 with Onnx "
         )
-        if sys.platform == "linux":
-            msg += "1.2.5 or "
         msg += "1.2.7."
```

Comment:
Since we no longer have branching logic based on OS, would you mind folding this into a single str to avoid the extra bin op and alloc?
```py
"Instead consider using Python 3.8 or 3.9 with RedisAI 1.2.7"
#                                              ^^^^^^^^^^^^^
# Also could fix this `Onnx` vs `RAI` typo, while your here?
```

Reply:
OK, wait, here there is something I don't understand (and I think @mellis13 also mentioned this): if a user is here, RedisAI 1.2.7 is being used (we do not support other versions), but the problem is that ONNX is not compatible with Python. Should we put the ONNX version (1.11) instead of the RedisAI version?

Reply:
@mellis13 @MattToast unifying warning message thread

Reply:
^^ Agreed! lets use that